content/09.response-to-reviewers.md

```diff
@@ -0,0 +1,183 @@
+
+**In this situation, a new word wouldn't be detected as you would need more papers over time to detect any differences.
+Arguably, it is unlikely that a word would have a semantic change in this context.
+As we found within our paper and previous works, a word needs to have significant usage for a change to occur.
+**
+
+
+>Q3: When detecting semantic changes, the cosine distance was used for comparing similarities between words. This method seems too easy and not that robust. Are there any other good ways for detecting?
+
+**New ways of detecting semantic change remain open for investigation.
+However, most work that detects semantic changes uses cosine similarity (1-cosine distance).
+Due to this work being the first within biomedical literature, we sought to keep the metric the same.
+**
+
+## Reviewer 2
+
+>This paper develops a method to evaluate semantic shifts by calculating annual and interannual changes using multiple integrated models. This method achieves good results, but there are some shortcomings in the paper. There are some suggestions for revision.
+>1.      The motivation is not clear. Please specify the importance of this paper.
+
+**We have updated our introduction to make our motivation for this work more clear.**
+```diff
+```
+
+>2.      Please highlight the contributions of this paper.
+
+**We have updated our introduction to make our contributions for this work more clear.**
+```diff
+```
+
+>3.      Most of references are out of date. Please discuss more recently published solutions, especially the solutions published in 2022.
+
+**We have updated our introduction to include more recent publications that pertain to this work.**
+
+>4.      In the method part, the authors’ description is not easy to understand. Have the authors considered using drawings to explain the method?
+
+**We have updated our methods section to include a graphical depiction of our data pipeline.**
+
+>5.      When detecting semantic changes across time, the authors mention the use of Cartesian products in calculating interannual changes. What are the advantages of using Cartesian products for the task of this paper?
+
+**The main advantage here is that the cartesian product allows us to directly compute the inter-year distances without the need for a statistical correction.
+Since we compute all possible combinations of model pairings, our sample size is sufficiently large enough to obtain inter-year distance estimates.**
+
+
+>6.      There is something unclear about Figure 1. There are five black dots in the second picture of Figure 1. What is the specific meaning of these five black dots? Please explain it to the author.
+
+**The black dots in Figure 1 are the individual word vectors obtained from their corresponding word2vec model.
```

Comment:
for which word? "Probiotics"? If so, I recommend adding that here.

Reply:
Also a good point to make a nod to your new figure which should make this process a lot clearer!